includes/free-trial/partners/class-wc-calypso-bridge-partner-stripe.php

```diff
@@ -0,0 +1,162 @@
+<?php
+
+/**
+ * WC Calypso Bridge Partner Stripe
+ *
+ *	@since   2.5.4
+ *	@version 2.5.4
+ *
+ * This file includes customizations for the sites that were created through /start-with/stripe on wordpress.com.
+ * woocommerce_onboarding_profile.partner must get 'stripe'
+ */
+class WC_Calypso_Bridge_Partner_Stripe {
+
+	/**
+	 * Class instance.
+	 *
+	 * @var WC_Calypso_Bridge_Partner_Stripe instance
+	 */
+	protected static $instance = false;
+
+	/**
+	 * Get class instance
+	 */
+	public static function get_instance() {
+		if ( ! self::$instance ) {
+			self::$instance = new self();
+		}
+
+		return self::$instance;
+	}
+
+	/**
+	 * Constructor.
+	 */
+	public function __construct() {
+		// Only for free trials.
+		if ( ! wc_calypso_bridge_is_ecommerce_trial_plan() ) {
+			return;
+		}
```

Comment:
Let's remove this condition, `$onboarding_profile['partner']` should be enough.

Reply:
I created a [follow-up task](https://github.com/Automattic/dotcom-forge/issues/8418) where we will work on improvements such as removing duplicated code.